search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -75,12 +76,83 @@ function displayTopResultsForCurrentTrend(countryCode) {
         countryData[0].averageSentiment + '<br>';
 
     for (let i = 0; i < results.length; i++) {
-      resultElement.innerHTML += '<a href=' + results[i].link + '>' +
-        results[i].htmlTitle + '</a><br>' + results[i].snippet + '<br>'
-        + 'Sentiment Score: ' + results[i].score + '<br>';
+      let searchResultIndex = i + 1;
+      resultElement.innerHTML += searchResultIndex + '. ' + '<a href=' + 
+          results[i].link + '>' + results[i].htmlTitle + '</a><br>' + 
+          results[i].snippet + '<br>' + 'Sentiment Score: ' + results[i].score + 
+          '<br>';
     }
     resultElement.innerHTML += '<i>Last updated on ' + date.toString() +
       '<i><br>';
   }
 }
 
+
+/** 
+ * Displays the sentiment chart in a country for current search trend on modal. 
+ * @param {string} countryCode Two letter country code for selected country.
+ */
+function displaySentimentChartForCurrentTrend(countryCode) {
+  let topic = getCurrentTrend;
+  let topicData = getCurrentCustomSearchData();
+  let date = new Date(topicData.timestamp);
+  let chartElement = document.getElementById('sentiment-chart-tab');
+  chartElement.innerHTML = '';
+
+  let countryData = topicData.countries
+      .filter(countries => countries.country === countryCode);
+
+  // Handle case where there are no results.
+  if (countryData.length === 0) {
+    chartElement.innerHTML += 'No results.<br><i>Last updated on ' +
+        date.toString() + '<i><br>';
+  } else {
+    let results = countryData[0].results;
+    drawSentimentChart(chartElement, results);
+    chartElement.innerHTML += 'Average Sentiment Score: ' + 
+        countryData[0].averageSentiment + '<br>';
+    chartElement.innerHTML += '<i>Last updated on ' + date.toString() +
+        '<i><br>';
+  }
+}
+
+google.charts.load('45', {'packages':['corechart']});
+/** Draws a sentiment chart and adds it to the sentiment chart modal tab. 
+ *  @param {Object} chartElement Tab element to update with the sentiment chart.
+ *  @param {Object} results Results to use to update the sentiment chart.
+ */
+function drawSentimentChart(chartElement, results) {
+  var sentimentDataArray = new Array();
+  sentimentDataArray.push(["Search Result", "Score", {role: "style"}]);
+  for (let i = 0; i < results.length; i++) {
+    let sentimentItem = [(i + 1).toString(), results[i].score];
+    if (results[i].score >= 0) {
+      sentimentItem.push('green');
```

Comment:
Maybe add constants to refer to these colors?
something like POSITIVE_COLOR and NEGATIVE_COLOR
Updating would be easier.